iam/api-client/custom_roles_test.py

```diff
@@ -71,7 +71,7 @@ def test_query_testable_permissions(capsys):
         "//cloudresourcemanager.googleapis.com/projects/" + GCLOUD_PROJECT
     )
     out, _ = capsys.readouterr()
-    assert "apigee" in out
+    assert "apigateway" in out
```

Comment:
I think it would make more sense to use "accessapproval", since it's the first service on the returned list.

"apigateway" is pretty close to the bottom, and if we add more permissions starting with "a," it might get bumped to the next page.

Reply:
Ah, so this fails because the permission wiped from the first page? I'd just check if there's a newline then.